### PR TITLE
Quizlet 2: Setup firebase

### PR DESCRIPTION
**Pull request by PTLam-Thryve**
Changes:
- Create Android and iOS projects in Firebase
- Setup Flutter project with Firebase

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
 import 'package:flutter/material.dart';
+import 'package:quizlet_clone/firebase_options.dart';
+
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
+  );
 
-void main() {
```
Comment (Megadebt) [inline lib/main.dart:3]: @PTLam-Thryve, please remove empty line before `runApp`

Comment (PTLam-Thryve) [reply lib/main.dart:3]: @Megadebt, done. Please recheck

**Review by Megadebt — APPROVED** [review]

**Comment by PTLam-Thryve** [comment]
@CodiumAI-Agent /describe

**Comment by PTLam-Thryve** [comment]
@CodiumAI-Agent /review

**Pull request merged**